README.md

```diff
@@ -1,39 +1,118 @@
-zimlib
-------
+ZIM library
+=============
 
-The `zimlib` library is the standard implementation of the ZIM
-specification.  It is a library which implements read and write
-methods for ZIM files.
+The ZIM library is the reference implementation for the openZIM file
+format. It's a solution to read and write ZIM file on many systems and
+architectures. More information about the ZIM format and the openZIM
+project at http://www.openzim.org/
 
-Use the zimlib in your own software --- for example, reader
-applications --- to make them ZIM-capable without the need having to
-dig too much into the ZIM file format.
+Disclaimer
+----------
 
-To build:
+This document assumes you have a little knowledge about software
+compilation. If you experience difficulties with the dependencies or
+with the ZIM libary compilation itself, we recommend to have a look to
+[kiwix-build](https://github.com/kiwix/kiwix-build).
+
+Preamble
+--------
+
+Although the ZIM library can be compiled/cross-compiled on/for many
+sytems, the following documentation explains how to do it on POSIX
+ones. It is primarly though for GNU/Linux systems and has been tested
+on recent releases of Ubuntu and Fedora.
+
+Dependencies
+------------
+
+The Kiwix library relies on many third parts software libraries. They
+are prerequisites to the Kiwix library compilation. Following
+libraries need to be available:
+
+* Z ................................................. http://zlib.net/
+(package zlib1g-dev on Ubuntu)
+* Bzip2 ......................................... http://www.bzip.org/
+(package libbz2-dev on Ubuntu)
+* Lzma ...................................... http://tukaani.org/lzma/
+(package lzma-dev on Ubuntu)
+* ICU ................................... http://site.icu-project.org/
+(package libicu-dev on Ubuntu)
+* Xapian ......................................... https://xapian.org/
+(package libxapian-dev on Ubuntu)
+
+These dependencies may or may not be packaged by your operating
+system. They may also be packaged but only in an older version. The
+compilation script will tell you if one of them is missing or too old.
+In the worse case, you will have to download and compile bleeding edge
+version by hand.
+
+If you want to install these dependencies locally, then use the ZIM
+library directory as install prefix.
```

Comment:
> If you want to install these dependencies locally, then use the ZIM library directory as install prefix.

If you want to install these dependencies locally, then ensure that meson (through pkg-config) will properly find them.
The easiest way to do that is to install them in the same directory as libzim will be installed.
Or you can set `PKG_CONFIG_PATH` environment variable to let pkg-config find the libraries.